frontend/app/Lightbox/NewLightbox.tsx

```diff
@@ -89,6 +97,7 @@ const NewLightbox:React.FC<RouteComponentProps> = (props) => {
     const [itemLimit, setItemLimit] = useState(300);
     const [basicSearchUrl, setBasicSearchUrl] = useState<string|undefined>(undefined);
     const [showingArchiveSpinner, setShowingArchiveSpinner] = useState(false);
+    const [filterString, setFilterString] = useState<string>("");
```

Comment:
just to note - `<string>` is not needed when the default value can only be a string (as in `""`)